src/ajax.js

```diff
@@ -25,10 +26,14 @@ function parseResponse(response) {
 }
 
 export default function ajax(method, url, opts = {}) {
-  const xhr = new XMLHttpRequest();
-
-  if (!('withCredentials' in xhr)) {
-    return ie9Ajax(...arguments);
+  // we need to check that we're not running in Node
+  // before we should check if this is ie9
+  if (!isNodeLikeEnvironment()) {
+    const xhr = new XMLHttpRequest();
+
+    if (!('withCredentials' in xhr)) {
```

Comment:
No... I'd need to look at this in the other PR. For this PR the main focus is on getting NPM up and running.